mathext/ell_complete.go

```diff
@@ -155,3 +155,168 @@ func CompleteE(m float64) float64 {
 
 	return 1
 }
+
+// CompleteB computes an associate complete elliptic integral of the 2nd kind, 0≤m≤1.
+// It returns math.NaN() if m is not in [0,1].
+//
+//	B(m) = \int_{0}^{\pi/2} {\cos^2\theta} / {\sqrt{1-m{\sin^2\theta}}} d\theta
+//
+// See: http://dx.doi.org/10.1016/j.cam.2014.12.038 for the computation method.
```

Comment:
Move implementation details inside the function. Could you enhance the citation? The linked article is 50 pages long. In particular, I don't see the numbers below in any of the tables.